src/applications/vaos/actions/newAppointment.js

```diff
@@ -270,6 +300,14 @@ export function updateFacilityPageData(page, uiSchema, data) {
           typeOfCareId,
         );
 
+        const availableFacilities = getEligibleFacilities(facilities);
+
+        // If no available facilities, fetch system details to display contact info
+        if (!availableFacilities?.length) {
```

Comment:
Do we need to check that these facility details are correct for the chosen facility? It looks like if I:

1. Update both `facilities_` json files to return empty arrays
2. Go to the facility page and choose one system
3. Click the other system
4. The alert message has old facility information

Reply:
Hmm.. The component doesn't seem to re-render when the props change.  I think it has something to do with being in the `ui:field`. Will need to look into it or go with your connected component suggestion.

Reply:
So I figured it out, and of course it was something so silly.  The content of the `<AlertBox/>` was placed in its children instead of the `content` prop. 🤦🏻‍♂️

Facility info should update properly now.

Reply:
Oh, really? That's very weird, those two ways of including the content should be equivalent.